### DISCUSSION TITLE
What if software have an "auto-update" feature ?

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/winget-cli#1379] [Q&A] [answered]
Hello,

I'm sorry, isn't really a documentation issue. I browsed the documentation but  I did not find my answer.

For example I install Firefox 90 with winget. Few days later, Firefox updates itself to 91. How is it going afterwards with winget?

- Community repo also updated to 91: everything is okay on Winget ?
- Community repo stayed to 90 : error ? Unknow ? 
- Winget no longer manages Firefox updates ?

Thank you !

**Comment by Papierjaeger** [comment, marked as answer]
If your version is higher than Community repo, nothing will happen.
You can test it with Krita, because the latest version of Krita is 4.4.7 but community repo is only at 4.4.5
